### PR TITLE
adding triggers for export settings

### PR DESCRIPTION
**Pull request by labhvam5**

### CODE REVIEW

File: apps/fyle/tasks.py [hunk 1]
```diff
@@ -81,14 +81,25 @@ def async_create_expense_groups(workspace_id: int, fund_source: List[str], task_
...
+                state=expense_group_settings.ccc_expense_state,
+                settled_at=last_synced_at if expense_group_settings.ccc_expense_state == 'PAYMENT_PROCESSING' else None,
                 filter_credit_expenses=True,
-                last_paid_at=last_synced_at if expense_group_settings.expense_state == 'PAID' else None
+                last_paid_at=last_synced_at if expense_group_settings.ccc_expense_state == 'PAID' else None
             )
 
+            if SOURCE_ACCOUNT_MAP['PERSONAL'] in source_account_type: 
+                reimbursable_expenses = platform.expenses.get(
+                    source_account_type=source_account_type,
```
Comment (Sravanksk) [inline apps/fyle/tasks.py:95]: same here `source_account_type` should only have PERSONAL, not both the `SOURCE_ACCOUNT_MAP` values

File: apps/workspaces/apis/export_settings/triggers.py [hunk 2]
```diff
@@ -0,0 +1,14 @@
+from apps.mappings.tasks import schedule_auto_map_employees
+from apps.workspaces.models import WorkspaceGeneralSettings
+
```
Comment (Sravanksk) [inline apps/workspaces/apis/export_settings/triggers.py:3]: please add 2 lines gap before any class - please fix this everywhere

File: tests/test_workspaces/fixtures.py [hunk 3]
```diff
@@ -25,6 +25,7 @@
         'id':2,
         'name':'Fyle For MS Dynamics Demo',
         'fyle_org_id':'orNoatdUnm1w',
+        'onboarding_state': 'CONNECTION',
```
Comment (ashwin1111) [inline tests/test_workspaces/fixtures.py:28]: Why is this added? the actual column to db is not added yet right?

File: tests/test_workspaces/test_apis/test_export_settings/test_views.py [hunk 4]
```diff
@@ -0,0 +1,36 @@
+import json
+from apps.workspaces.models import Workspace,WorkspaceGeneralSettings
+from .fixtures import data
+from tests.helper import dict_compare_keys
+
+
+def test_export_settings(api_client, test_connection):
+
+    workspace = Workspace.objects.get(id=1)
+    workspace.onboarding_state = 'EXPORT_SETTINGS'
```
Comment (ashwin1111) [inline tests/test_workspaces/test_apis/test_export_settings/test_views.py:10]: same

File: tests/test_workspaces/test_apis/test_export_settings/test_views.py [hunk 5]
```diff
@@ -0,0 +1,36 @@
...
+    response = api_client.put(
+        url,
+        data=data['export_settings'],
+        format='json'
+    )
+
+    assert response.status_code == 200
+
+    response = json.loads(response.content)
+    assert dict_compare_keys(response, data['export_settings_response']) == [], 'workspaces api returns a diff in the keys'
```
Comment (ashwin1111) [inline tests/test_workspaces/test_apis/test_export_settings/test_views.py:28]: export settings api instead of workspaces

**Review by ashwin1111 — APPROVED** [review]
